pb/mgmt_pb/decoder.go

```diff
@@ -300,7 +300,7 @@ func (d Decoder) Decode(msg *channel2.Message) ([]byte, bool) {
 }
 
```

Comment:
We need to make sure we update the `Decoder` for the mgmt channel, and for anything new traversing the data plane or the control plane.